slices/ucf.yaml

```diff
@@ -0,0 +1,22 @@
+package: ucf
+
+# uses /usr/share/ucf/ucf_helper_functions.sh for some debhelper functions
+# during install and removal.
+slices:
+  base:
+    essential:
+      - ucf_bins
+      - ucf_config
+
+  bins:
+    essential:
+      - sensible-utils_base
```

Comment:
We do not need the mime file from `sensible-utils`.

```suggestion
      - sensible-utils_bins
```

Reply:
nice catch, updated this